### PR TITLE
Daylight: Checkbox & Radio

### PR DESCRIPTION
**Pull request by margaree**
I investigated combining these into 1 page, but they both have quite a bit of content and so I decided to leave them separate at least for now.

Checkbox is fairly straightforward. Radio is a bit more complex, and so I took a strategy similar to that used with table where I used example components for demos, and added a couple additional demos.

There are a couple small tweaks to input-date-time and table which I put in here as well.

### CODE REVIEW

File: components/inputs/docs/input-checkbox.md [hunk 1]
```diff
@@ -24,43 +72,49 @@ The `<d2l-input-checkbox>` element can be used to get a checkbox and optional vi
...
 
 | Attribute | Description |
-|--|--|
+|---|---|
 | `aria-label` | Use when text on checkbox does not provide enough context |
 | `description` | Use when label on input does not provide enough context. |
 
+### Usage
+
 **Methods:**
```
Comment (GeorgePlukov) [inline components/inputs/docs/input-checkbox.md:97]: Are we including bold titles like this within daylight pages? Maybe we remove the `Methods` title or switch the H3 above it to `Usage Methods`

File: components/inputs/docs/input-radio.md [hunk 2]
```diff
@@ -1,19 +1,52 @@
...
 
+## Best Practices
+<!-- docs: start best practices -->
+<!-- docs: start dos -->
+* Use as an input for traditional forms
+* Use when there are 3 or more mutually exclusive options
+<!-- docs: end dos -->
+
+<!-- docs: start donts -->
+* Don’t use 2 radios if a 1 checkbox works better
```
Comment (GeorgePlukov) [inline components/inputs/docs/input-radio.md:31]:
```suggestion
* Don’t use two radio buttons if a checkbox works better
```

File: components/inputs/docs/input-radio.md [hunk 3]
```diff
@@ -66,10 +99,28 @@ class MyElem extends LitElement {
...
+  import '@brightspace-ui/core/components/inputs/demo/input-radio-solo-test.js';
+</script>
+<d2l-test-input-radio-solo></d2l-test-input-radio-solo>
+```
+
+## Radio Spacer [d2l-input-radio-spacer]
 
 To align related content below radio buttons, the `d2l-input-radio-spacer` element can be used:
 
+<!-- docs: demo display:block -->
```
Comment (GeorgePlukov) [inline components/inputs/docs/input-radio.md:116]: Why do these need `display: block`?

File: components/inputs/docs/input-checkbox.md (hunk 1 above)
Comment (margaree) [reply components/inputs/docs/input-checkbox.md:97]: I believe we'd discussed having a separate "usage" or "how to use" section in components where necessary and so calling it "usage" would keep that consistent, and then here within that is methods. I'll confirm with Jeff though.

File: components/inputs/docs/input-radio.md (hunk 3 above)
Comment (margaree) [reply components/inputs/docs/input-radio.md:116]: I feel like there was some weird spacing happening, but yeah they do seem fine. I'll re-test and remove.

Comment (GeorgePlukov) [reply components/inputs/docs/input-radio.md:116]: Yeah no problem to leave it, just expected these to work well with `display: flex`

**Review by GeorgePlukov — APPROVED** [review]

**Pull request merged**